### PR TITLE
Add Pool2 on ethereum, modify logo Plz!

### PR DESCRIPTION
**Pull request by copyNdpaste**
##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
image

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/cryptoyieldfocus/index.js exports TVL: 
                
 
 
avalanche-staking         0
avalanche-pool2           12.20694478871922
avalanche                 9.40 k
staking                   0
pool2                     12.20694478871922

total                    9.40 k

**Comment by llamatester** [comment]
The adapter at projects/elysia/index.js exports TVL: 
                
 
 
ethereum-pool2            416.37 k
bsc                       724.33 k
ethereum-staking          549.42 k
bsc-staking               269.51 k
ethereum                  3.10 M
pool2                     416.37 k
staking                   818.94 k

total                    3.82 M

File: projects/cryptoyieldfocus/index.js [hunk 1]
```diff
@@ -30,16 +35,24 @@ const pool2 = async (timestamp, ethBlock, chainBlocks) => {
...
+  await sumTokensAndLPsSharedOwners(
+    balances,
+    pool2s.map((p) => [p, true]),
+    [STAKING_CONTRACT],
+    chainBlocks.avax,
+    "avax",
+    transformAddress
+  );
+
+  console.log(`@@@@`, JSON.stringify(balances));
```
Comment (g1nt0ki) [inline projects/cryptoyieldfocus/index.js:47]: can you remove this line

Comment (copyNdpaste) [reply projects/cryptoyieldfocus/index.js:47]: ok

File: projects/elysia/index.js [hunk 2]
```diff
@@ -2,62 +2,135 @@ const sdk = require("@defillama/sdk");
...
+      params: addresses.elStaking,
+      abi: "erc20:balanceOf",
+      block,
+    })
+  ).output;
 
-    sdk.util.sumSingleBalance(balances, addresses.el, elStakingValue);
-    sdk.util.sumSingleBalance(balances, addresses.dai, reserves[0].totalDeposit);
-    sdk.util.sumSingleBalance(balances, addresses.usdt, reserves[1].totalDeposit);
+  console.log(`elStakingValue ${elStakingValue}`);
```
Comment (g1nt0ki) [inline projects/elysia/index.js:59]: please remove these console log lines as well

File: projects/elysia/index.js [hunk 3]
```diff
@@ -2,62 +2,135 @@ const sdk = require("@defillama/sdk");
...
+  sdk.util.sumSingleBalance(balances, addresses.el, elStakingValue);
```
Comment (g1nt0ki) [inline projects/elysia/index.js:59]: @waynebruce0x the staking should not be part of total TVL right?

**Comment by llamatester** [comment]
The adapter at projects/cryptoyieldfocus/index.js exports TVL: 
                
 
 
avalanche-staking         0
avalanche-pool2           11.976333224096352
avalanche                 9.20 k
staking                   0
pool2                     11.976333224096352

total                    9.20 k

**Comment by llamatester** [comment]
The adapter at projects/elysia/index.js exports TVL: 
                
 
 
ethereum-pool2            416.93 k
bsc                       721.77 k
bsc-staking               269.52 k
ethereum-staking          549.42 k
ethereum                  3.10 M
pool2                     416.93 k
staking                   818.95 k

total                    3.82 M

**Comment by llamatester** [comment]
The adapter at projects/elysia/index.js exports TVL: 
                
 
 
ethereum-pool2            416.93 k
bsc-staking               269.52 k
ethereum-staking          549.42 k
bsc                       721.77 k
ethereum                  3.10 M
pool2                     416.93 k
staking                   818.95 k

total                    3.82 M

**Comment by copyNdpaste** [comment]
`cryptoyieldfocus/index.js` adjusting was mistake, so i removed the mistakenly formatted codes

File: projects/elysia/index.js (hunk 3 above)
Comment (copyNdpaste) [reply projects/elysia/index.js:59]: @g1nt0ki so non governance token `el` was added
image

**Comment by waynebruce0x** [comment]
I didnt include the pool2 because its not part of your dapp, its just counting uniswap pools, which is their TVL not yours. If you get staked LP tokens on your UI we can add it as pool2. We can add the bsc staking though

**Comment by copyNdpaste** [comment]
Why raised tvl key problem? i wrote tvl on ethereum, bsc exports.
`api.llama.fi/coins failed` ... is this associated with coingecko? i think this codes worked few hours ago. but not now haha. How to solve this? help me. thank u

image

**Comment by copyNdpaste** [comment]
Why raised tvl key problem? i wrote tvl on ethereum, bsc exports.
`api.llama.fi/coins failed` ... is this associated with coingecko? i think this codes worked few hours ago. but not now haha. How to solve this? help me. thank u

image

File: projects/elysia/index.js (hunk 3 above)
Comment (g1nt0ki) [reply projects/elysia/index.js:64]: el is still native token, so doubt it should be included

**Comment by llamatester** [comment]
The adapter at projects/elysia/index.js exports TVL: 
                
 
 
bsc-staking               276.36 k
bsc                       782.22 k
ethereum-staking          562.76 k
ethereum                  3.10 M
staking                   839.12 k

total                    3.88 M

**Comment by waynebruce0x** [comment]
https://github.com/DefiLlama/DefiLlama-Adapters/pull/2007

**Pull request closed without merging**